how_to_add_new_hotspot.md

```diff
@@ -0,0 +1,142 @@
+# How To Add A New Hotspot Into Nebra Firmware
+
+The Nebra firmware stack has been designed to support various different architectures and vendors. Hence it is very flexible by design. But this also gives the developers a responsibility about supplying correct hardware configuration.
+
+This configuration information has to be supplied in different places of the stack. Let's review them one by one.
+
+## Checklist for adding a new hotspot type
+
+* Add variant into `variant_definitions` in hm-pyhelper repository
+* Create custom inputs (e.g. additional buttons) and outputs (e.g. additional LED's) in the new variant and update the logic in other repositories if necessary.
+* Release and publish it via CI/CD pipeline
+* Create related production fleets as "helium-<freq>-<device_type>"
+* Edit configuration parameters of the fleet
+* Add `VARIANT` and `FREQ` environment variables into fleet variables
+* Update automation steps in this repository for newly created fleets
+* Add sentry environment variables into fleet variables
+
+## Defining the variant in hm-pyhelper project and releasing it
```

Comment:
we should probably mention somewhere that after adding the variant to hm-pyhelper and releasing it we will need to bump the hm-pyhelper version number in hm-pktfwd, hm-diag and hm-config and depending on the exact parameters of the miner (does the i2c-bus and i2c address match an existing config), we may need to also update the docker.config file.<device> file in hm-miner and hm-block-tracker

perhaps we could link to some example PRs of all of these different things as well

Reply:
Added many points but I'm against using I2C config in other places. Variant definition has to be enough. If not, it means waiting for us to fix.

hm-miner and hm-block-tracker are mentioning only to helium key spot on the ECC, not the I2C device. Not added reference for them.

Added example pull request for such case.